packages/back-end/src/models/DataSourceModel.ts

```diff
@@ -88,6 +88,13 @@ export async function getDataSourcesByOrganization(
     context.permissions.canReadMultiProjectResource(ds.projects)
   );
 }
+// WARNING: This does not restrict by organization
+export async function _dangerousGetAllGrowthbookClickhouseDataSources() {
```

Comment:
I thought we wanted to cache this? Maybe at least a in-code cache using a dangling `let` variable.

Reply:
Yeah that's what I had initially but Jeremy suggested letting the ingestion endpoint handle the caching rather than having two caching layers.

Reply:
Yeah, it's using an index so will be super fast.  Dealing with cache invalidation in a horizontally scaled deployment is not fun and not worth it here.

Reply:
Word.